getting-started/cloud-multi-node.md

```diff
@@ -0,0 +1,250 @@
+
+Once you have the **password** and **host** for each data node
+Service, connect to the access node using the `tsdbadmin` user. On Timescale Cloud
+this is easily accomplished by copying the **Service URI** (after clicking 
+`CLICK_TO:REVEAL_PASSWORD`). It should look something like the following, but
+with different ports, password and Service URI.
+
+Connect using `psql`:
+
+```bash
+psql postgres://tsdbadmin:abcd1234@dn1-cloud-demo-support-50d0.a.timescaledb.io:12345/defaultdb?sslmode=require
+```
+
+Then add a data node as follows.
+
+```sql
+SELECT add_data_node('dn1', host => 'your_DN1_hostname', port => 12345
+	password => 'tsdbadmin_user_password_for_DN1');
+```
+
+To list added data nodes we can run `\des+` command if using `psql`. 
+
+```bash
+tsdb=> \des+
+List of foreign servers
+-[ RECORD 1 ]--------+------------------------------------------------------------------------------------
+Name                 | dn1
+Owner                | multinode
+Foreign-data wrapper | timescaledb_fdw
+Access privileges    | 
+Type                 | 
+Version              | 
+FDW options          | (host 'dn1-cloud-demo-support-50d0.a.timescaledb.io', port '12345', dbname 'defaultdb')
+Description          | 
+```
+
+### Step 3b: Add a User Mapping for each data node [](step3b)
+
+Now we can create a `USER MAPPING` that will enable communication between the 
+access node and data node.
+
+```SQL
+CREATE USER MAPPING FOR tsdbadmin SERVER dn1 OPTIONS (user 'tsdbadmin', password 'tsdbadmin_user_password_for_DN1');
+```
+
+Repeat these steps for each additional data node that you want to add to the 
+cluster. **Always invoke these commands from the access node!**
+
+### Step 4: Create a distributed hyptertable [](step4)
+
+Finally, we can create a distributed hypertable and add data to verify that everything is
+set up and working correctly.
+
+```SQL
+-- Create your regular table
+CREATE TABLE sensor_data (
+  time TIMESTAMPTZ NOT NULL,
+  sensor_id INTEGER,
+  temperature DOUBLE PRECISION,
+  cpu DOUBLE PRECISION
+);
+
+-- Convert the table to a distributed hypertable
+SELECT create_distributed_hypertable('sensor_data', 'time', 'sensor_id');
+
+-- Insert some test data to verify you are error free
+INSERT INTO sensor_data VALUES  ('2020-12-09',1,32.2,0.45);
+```
+
+One major point to recognize about the SQL above is the declaration of a partition 
+column (`sensor_id`) for the distributed hypertable. This is intentional, and 
+**recommended**, for distributed hypertable setups. Previously, with regular, 
+single-node hypertables, there was often little benefit in specifying a partition 
+key when creating the hypertable. With distributed hyptertables, however, adding
+a partition key is essential to ensure that data is distributed across data nodes 
+on something more than just time. Otherwise, all data for a specific time range 
+will go to one chunk on one node, rather than being distributed across each data 
+node for the same time range.
```

Comment:
```suggestion
a partition key is essential to ensure that data is efficiently distributed across data nodes. 
Otherwise, all data for a specific time range will go to one chunk on one node, rather than 
being distributed across all available data nodes for the same time range.
```